### DISCUSSION TITLE
Spotbug Html report

### DISCUSSION THREAD

**Discussion by nikithagm** [spotbugs/spotbugs#2347] [Q&A]
SPotbug is integrated to Azuredevsecops pipelines using maven project. Once scanning is done ,xml reports are generated but not the html reports in pipeline artifacts. 
Added below entries in pom.xml file under configurations
                          
![Capture](https://user-images.githubusercontent.com/124684336/217651927-e85f29b6-c41b-41a5-823f-1e8a07e83baa.PNG)

As i said with this only xml report is genrated. Please do answer how to enable html report in pipeline artifacts for maven project.

**Comment by hazendaz** [comment]
@nikithagm This option did not exist in maven plugin until 4.7.3.1, bump your version up to 4.7.3.2 (latest).
